Tools/buildAllDefaultJaspModules.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env Rscript 
+
+args = commandArgs(trailingOnly=TRUE)
+
+# the args thing doesnt seem to work but the defaults are probably what you want anyway. otherwise plerase commit the fix ;)
+doTest  = FALSE
+doClone = TRUE
+
+sapply(args, function(arg) { if(arg == "test")  { doTest = TRUE; }})
+sapply(args, function(arg) { if(arg == "clone") { doClone = TRUE; }})
+
+modules = c(
+"jaspDescriptives",
+"jaspTTests",
+"jaspAnova",
+"jaspRegression",
+"jaspFrequencies",
+"jaspFactor",
+"jaspAudit",
+"jaspBain",
+"jaspNetwork",
+"jaspMachineLearning",
+"jaspMetaAnalysis",
+"jaspSem",
+"jaspSummaryStatistics",
+"jaspMixedModels",
+"jaspDistributions",
+"jaspEquivalenceTTests",
+"jaspJags",
+"jaspReliability",
+"jaspVisualModeling",
+"jaspBFF",
+"jaspBfpack",
+"jaspEsci"
+)
+
+if(doTest) modules = rbind(modules, "jaspTestModule")
```

Comment:
But how we can just build `jaspDescriptives` and `jaspTestModule` for a test build, this seems build all + test module.

Reply:
Yeah, it looks like we'd need to adjust this manually each time...

I also use a custom compiled version of R, and this will not work with that out of the box.

Reply:
The "build all" script is just for convenience, and to clone stuff.

But if you have your clones in a folder you can also just run  `./buildModuleBundlesLocally.sh /path/to/modules/*`
Then if that folder just contains descriptives and testmodule it will install those.

You can also run it on the result of "build all".

I thought this would be obvious from the scripts...
Should I write more comments in there?
Or something?

Reply:
And the beautiful thing is, you only need to do this if you actually want to update the modules.
Not if you just want to clear your buildfolder or something.

Reply:
> I also use a custom compiled version of R, and this will not work with that out of the box.

The bundles for flatpak are build in the flatpak environment - jasp-desktop itself.

Reply:
> occasionally

basically always when developing. Also, `RScript` points to the system `RScript`, but that is not what JASP uses or builds with. This would lock me (and anybody else developing on Linux) into always using the system version of R.

Reply:
https://github.com/jasp-stats/jasp-desktop/pull/5899
Uses the R version in cmake and a separate lib to install the tool (jaspModuleTools)

Reply:
Aah yeah I had just been using the systemversion the whole time tbh

Reply:
nope and actually my script also ignores the args so you just have to install the testmodule manually or clone it in the modules folder or something